src/Transaction.jsx

```diff
@@ -0,0 +1,225 @@
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.to ?  <List.Item>
+                          <List.Content>
+                            To
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Hash value={this.state.transaction.to}></Hash>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.ether ?  <List.Item>
+                          <List.Content>
+                            Ether
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Rspan>{formatBalance(this.state.transaction.value)}</Rspan>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.condition ? <List.Item>
+                          <List.Content>
+                            Condition
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Rspan>{formatCondition(this.state.transaction.condition)}</Rspan>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.creates ? <List.Item>
+                          <List.Content>
+                            Creates
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Hash value={this.state.transaction.creates}></Hash>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.gas ? <List.Item>
+                          <List.Content>
+                            Gas
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Rspan>{this.state.transaction.gas.toString(10)}</Rspan>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.gasPrice ? <List.Item>
+                          <List.Content>
+                            Gas Price
+                          </List.Content>
+                          <List.Content floated='right'>
+                            <Rspan>{formatBalance(this.state.transaction.gasPrice)}</Rspan>
+                          </List.Content>
+                        </List.Item> : "" }
+                        {this.props.input ? <List.Item>
+                          <List.Content>
+                            Input Data
+                          </List.Content>
+                          <List.Content floated='right'>
+                            {/* Input is raw byte data */}
+                            <Hash value={this.state.transaction.input}></Hash>
```

Comment:
Better to format as raw bytes; ideally it would eventually parse ABIs, but that's definitely for version 2 :-)